packages/react-storage/src/components/StorageManager/types.ts

```diff
@@ -58,10 +56,10 @@ export interface StorageManagerProps {
    */
   acceptedFileTypes?: string[];
   /**
-   * @deprecated
```

Comment:
Why are we removing the deprecation warning?

Reply:
Let's chat about this offline, there are pros and cons both ways.